oncue-agent/src/main/java/oncue/agent/AbstractAgent.java

```diff
@@ -20,6 +20,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.function.Predicate;
```

Comment:
This class is from JDK 8. So the file is compiled with Java 7 but linked against JDK 8 jars?